task/fossa-scanner/0.1/fossa-scanner.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fossa-scanner
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, fossa
+    tekton.dev/displayName: "fossa scanner"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    fossa-cli is a zero-configuration polyglot dependency analysis tool. You can point fossa CLI at any codebase or build,
+    and it will automatically detect dependencies being used by your project.
+
+    This task can be used to scan for dependency analysis, license scanning, vulnerability scanning,
+    attribution report generation, and more on the source code in standalone mode.
+  workspaces:
+    - name: source
+      description: A workspace that contains the source code to scan.
+  params:
+    - name: FOSSA_API_KEY
+      description: API key to be passed to fossa cli.
+      type: string
+    - name: FOSSACLI_IMAGE
+      default: docker.io/nistal97/fossa-cli@sha256:b3dd65d1297e9436b9e0f176c355ea1fa9c6295d09f79032c830b6e7e3376293
+      description: fossa scanner image to be used
+      type: string
+    - name: FOSSA_ENDPOINT
+      description: fossa server endpoint to connect
+      default: https://fossa.com/
+      type: string
+    - name: FOSSA_TELEMETRY_SCOPE
+      description: scope of fossa analyze telemetry, could be "full" or "off"
+      type: string
```

Comment:
should we add a default value for `FOSSA_TELEMETRY_SCOPE` so that it can be made optional to the users?

Reply:
yes we can provide a default value , but to me i would like everyone to pay attention to this as it's important to build speed/performance, so better to let them notice and decide.  And it's a very easy configure.  let me know your throughts.